Gemfile

```
@@ -1,10 +1,10 @@
 source "https://rubygems.org"
 
-gem "jekyll", "~> 3.4"
+gem "jekyll", "~> 4.4"
 gem "jekyll-paginate", "~> 1.1"
-gem "jekyll-feed", "~> 0.9"
-gem "jekyll-sitemap", "~> 1.0"
-gem "jekyll-redirect-from", "~> 0.12"
-gem "classifier-reborn"
+gem "jekyll-feed", "~> 0.17"
```

RuboCop (1.76.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `jekyll-feed` should appear before `jekyll-paginate`.

(Bundler/OrderedGems)